File: docs/components_page/components/buttons/group.py
import dash_bootstrap_components as dbc
import dash_html_components as html

buttons = html.Div(
    [
        html.H4("Button group"),
        html.Div(
            dbc.ButtonGroup(
                [
                    dbc.Button("Primary", color="primary"),
                    dbc.Button("Secondary", color="secondary"),
                    dbc.Button("Success", color="success"),
                    dbc.Button("Warning", color="warning"),
                    dbc.Button("Danger", color="danger"),
                    dbc.Button("Info", color="info"),
                ]
            )
        ),
    ]
)

File: components_page/components/card/layout/group.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

cards <- dbcCardGroup(
  list(
    dbcCard(
      dbcCardBody(
        list(
          htmlH5("Card 1", className = "card-title"),
          htmlP(
            paste(
              "This card has some text content, which is a little",
              "bit longer than the second card."
            ),
            className = "card-text",
          ),
          dbcButton(
            "Click here",
            color = "success", className = "mt-auto"
          )
        )
      )
    ),
    dbcCard(
      dbcCardBody(
        list(
          htmlH5("Card 2", className = "card-title"),
          htmlP(
            "This card has some text content.",
            className = "card-text",
          ),
          dbcButton(
            "Click here",
            color = "warning", className = "mt-auto"
          )
        )
      )
    ),
    dbcCard(
      dbcCardBody(
        list(
          htmlH5("Card 3", className = "card-title"),
          htmlP(
            paste(
              "This card has some text content, which is longer",
              "than both of the other two cards, in order to",
              "demonstrate the equal height property of cards in a",
              "card group."
            ),
            className = "card-text",
          ),
          dbcButton(
            "Click here",
            color = "danger", className = "mt-auto"
          )
        )
      )
    )
  )
)
